LivingCity/traffic/b18TrafficSP.cpp

```diff
@@ -277,7 +365,21 @@ const std::vector<abm::graph::edge_id_t> B18TrafficSP::loadPrevPathsFromFile(
   return const_paths_SP;
 }
 
+//merge multi path into one path
```

Comment:
TODO: When we are outputting the things from the simulator, we need to change the things to be output in people and route file.